src/components/Banner/BaseAnnouncementBanner/styled.tsx

```diff
@@ -0,0 +1,84 @@
+import walletBannerPhoneImageSrc from 'assets/images/wallet_banner_phone_image.png'
+import { BaseButton } from 'components/Button'
+import { OpacityHoverState } from 'components/Common'
+import Row from 'components/Row'
+import { X } from 'react-feather'
+import styled from 'styled-components'
```

Comment:
caused by https://github.com/Uniswap/interface/pull/7056

```suggestion
import styled from 'styled-components/macro'
```